alfasim_sdk/hook_specs_gui.py

```diff
@@ -76,6 +74,45 @@ def alfasim_get_status(ctx):
     """
 
 
+@hookspec
+def alfasim_configure_fields():
+    """
+    Configure new fields to be added in ALFAsim's hydrodynamic model.
+    Example:
+
+    return [
+        AddField(name='plugin_continuous_field'),
+        AddField(name='plugin_dispersed_field')
+    ]
+    """
+
+
+@hookspec
+def alfasim_configure_layers():
+    """
+    Configure new or update layers for ALFAsim's hydrodynamic model.
+    Example:
+
+    return [
+        AddLayer(name='plugin_layer', fields=['plugin_continuous_field', DROPLET_FIELD], continuous_field='plugin_continuous_field'),
+        UpdateLayer(name=WATER_FIELD, additional_fields=['plugin_dispersed_field']),
+    ]
+    """
+
+
+@hookspec
+def alfasim_configure_phases():
+    """
+    Configure new or update phases for ALFAsim's hydrodynamic model.
+    Example:
+
+    return [
+        AddPhase(name='plugin_phase', fields=['plugin_continuous_field', DROPLET_FIELD], primary_field='plugin_continuous_field'),
```

Comment:
I don't think add a phase with pre-existing fields is a good idea. If possible, please remove `DROPLET_FIELD` from example.

Reply:
I need to merge this PR to fix the master on ALFAsim, I created an issue for us to deal with it later, it's okay for you?